SIM-ONE Training/simone_transformer/simone_model.py

```
@@ -1,13 +1,127 @@
+        # Feedforward with residual
+        ff_out = self.ff(x)
+        x = self.norm2(x + ff_out)
+
+        return x
+
+
+# Aliases for backward compatibility
+EnhancedSIMONEModel = SIMONEModel
+EnhancedSIMONEBlock = SIMONEBlock
 
 __all__ = ["SIMONEModel", "SIMONEBlock", "EnhancedSIMONEModel", "EnhancedSIMONEBlock", "GovernanceAggregator"]
```

Ruff (0.13.1)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)